packages/vue-components/src/panels/MinimalPanel.vue

```diff
@@ -3,7 +3,7 @@
     <div v-show="localMinimized" class="morph">
       <button class="morph-display-wrapper btn card-title morph-title" @click="open()">
         <slot name="_alt">
-          <slot name="header"></slot>
```

Comment:
hmm, is this needed for minimal panels? (the header already automatically hides when expanded)

Reply:
Yup this is not needed, thanks!